### PR TITLE
Set volume background

### PR DESCRIPTION
**Pull request by Arunshaik2001**

### CODE REVIEW

File: lib/src/hmssdk.dart [hunk 1]
```diff
@@ -24,7 +24,7 @@ import '../hmssdk_flutter.dart';
 /// **Broadcast** - A local peer can send any message/data to all remote peers in the room
 ///
 /// HMSSDK has other methods which the client app can use to get more info about the Room, Peer and Tracks
-class HMSSDK {
```
Comment (ygit) [inline lib/src/hmssdk.dart:27]: why `WidgetsBindingObserver` is this added?

File: example/lib/meeting/meeting_page.dart [hunk 2]
```diff
@@ -713,6 +724,11 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
   void didChangeAppLifecycleState(AppLifecycleState state) {
     super.didChangeAppLifecycleState(state);
     if (state == AppLifecycleState.resumed) {
+      _meetingStore.audioTracks.forEach((element) {
```
Comment (ygit) [inline example/lib/meeting/meeting_page.dart:727]: the way we have to do set volume is in AppLifecycleState.resumed > call hmsSDK's getPeers function > then iterate over all peer.audioTrack & peer.auxiliaryTrack > then set volume: 10.0
white_check_mark
eyes
raised_hands
don't add another audioTracks array in meetingStore

**Review by ygit — CHANGES_REQUESTED** [review]

File: lib/src/hmssdk.dart (hunk 1 above)
Comment (Arunshaik2001) [reply lib/src/hmssdk.dart:27]: removing

File: example/lib/meeting/meeting_page.dart [hunk 3]
```diff
@@ -276,7 +285,7 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
                 : Scaffold(
                     resizeToAvoidBottomInset: false,
                     appBar: AppBar(
-                      title: Text(Constant.meetingCode),
+                      title: Text(widget.roomId),
```
Comment (ygit) [inline example/lib/meeting/meeting_page.dart:288]: it should be `Text(Constant.meetingCode)`
why is this being changed?

File: example/lib/meeting/meeting_store.dart [hunk 4]
```diff
@@ -398,11 +398,18 @@ abstract class MeetingStoreBase extends ChangeNotifier
     }
 
     if (track.kind == HMSTrackKind.kHMSTrackKindAudio) {
+      int indexAudio = peerTracks.indexWhere((element) => element.peerId == peer.peerId);
```
Comment (ygit) [inline example/lib/meeting/meeting_store.dart:401]: remove `audioTracks` & this logic of adding to audioTracks

File: lib/src/hmssdk.dart [hunk 5]
```diff
@@ -5,7 +5,7 @@
 ///All methods related to meeting, preview and their listeners are present here.
 
 // Project imports:
-
+import 'package:flutter/material.dart';
```
Comment (ygit) [inline lib/src/hmssdk.dart:8]: remove this import

File: example/lib/meeting/meeting_page.dart [hunk 6]
```diff
@@ -238,24 +259,12 @@ class _MeetingPageState extends State<MeetingPage> with WidgetsBindingObserver {
         }
         break;
       case 9:
-        if (_meetingStore.hasHlsStarted) {
```
Comment (ygit) [inline example/lib/meeting/meeting_page.dart:241]: why is this logic removed?

**Review by ygit — APPROVED** [review]

**Pull request merged**